src/packet_analysis/protocol/ip/SessionAdapter.h

```diff
@@ -2,13 +2,90 @@
 
 #pragma once
 
+#include <memory>
+
 #include "zeek/analyzer/Analyzer.h"
+#include "zeek/iosource/Packet.h"
 
 namespace zeek::analyzer::pia {
 class PIA;
 }
 
-namespace zeek::packet_analysis::IP {
+namespace zeek::packet_analysis {
+
+/**
+ * Indicator for a TapAnalyzer to determine what will happen to a packet.
+ */
+enum class TapVerdict : uint8_t {
```

Comment:
Drive-by comment here. How about `TapAction`?

Reply:
As in DM, going for `PacketAction`